lib/archethic_web/controllers/api/transaction_controller.ex

```diff
@@ -119,4 +123,73 @@ defmodule ArchEthicWeb.API.TransactionController do
         |> render("400.json", changeset: changeset)
     end
   end
+
+  def origin_key(conn, %{"origin_public_key" => origin_public_key}) do
+    with {:ok, origin_public_key} <- Base.decode16(origin_public_key, case: :mixed),
+         <<_curve_id::8, origin_id::8, _rest::binary>> <- origin_public_key,
+         {first_origin_family_public_key, _} <-
+           SharedSecrets.get_origin_family_from_origin_id(origin_id)
+           |> SharedSecrets.get_origin_family_seed()
+           |> Crypto.derive_keypair(0),
+         {:ok, tx} <-
+           Crypto.derive_address(first_origin_family_public_key)
+           |> TransactionChain.get_last_transaction(data: [:ownerships]),
+         ownership when ownership != nil <-
+           Enum.find(tx.data.ownerships, fn ownership ->
+             Ownership.authorized_public_key?(ownership, origin_public_key)
+           end) do
+      res = %{
+        encrypted_origin_private_key: Base.encode16(ownership.secret),
```

Comment:
maybe rename it to: `encrypted_origin_private_keys` to support list of keys

Reply:
Do the encrypted secred key match for multiple secrets ?
Do I have a response like
```json
{
 "encrypted_origin_private_keys": ["pKey1", "pKey2", "pKey3"],
 "encrypted_secret_key": "EKey"
}
```
or 
```json
[
 {
  "encrypted_origin_private_key": "pKey1",
   "encrypted_secret_key": "EKey1"
 },
 {
  "encrypted_origin_private_key": "pKey2",
   "encrypted_secret_key": "EKey2"
 }
]
```